### PR TITLE
Add help message for DEL (ERASE) command

### PR DESCRIPTION
**Pull request by Wengier**
This adds help message for the DEL (ERASE) command, following the style of commands like VER. The command "DEL /?" will now show actual help messages for this command.

### CODE REVIEW

File: src/shell/shell.cpp [hunk 1]
```diff
@@ -714,6 +714,21 @@ void SHELL_Init() {
 	        "REN [drive:][path]filename1 filename2.\n\n"
 	        "Note that you can not specify a new drive or path for your destination file.\n");
 	MSG_Add("SHELL_CMD_DELETE_HELP","Removes one or more files.\n");
+	MSG_Add("SHELL_CMD_DELETE_HELP_LONG", "Usage:\n"
+	        "  \033[32;1mdel\033[0m \033[36;1mPATTERN\033[0m\n"
+	        "  \033[32;1merase\033[0m \033[36;1mPATTERN\033[0m\n"
+	        "\n"
+	        "Where:\n"
+	        "  \033[36;1mPATTERN\033[0m is a file name or pattern that contains wildcards (* or ?),\n"
```
Comment (kcgen) [inline src/shell/shell.cpp:722]: Where PATTERN can be either an exact filename or an inexact filename using one or more wildcards, which are the asterisk (*) that represents any sequence of one or more characters, and the question mark (?) that represents any single character.  For example ...

File: src/shell/shell.cpp [hunk 2]
```diff
@@ -714,6 +714,21 @@ void SHELL_Init() {
...
+	        "          such as \033[36;1mfile.txt\033[0m, or \033[36;1m*.bat\033[0m.\n"
+	        "\n"
+	        "Notes:\n"
+	        "  Be careful with deleting files containing wildcards, especially \033[36;1m*.*\033[0m.\n"
```
Comment (kcgen) [inline src/shell/shell.cpp:726]: Instead of Notes:, let's use Warning:

File: src/shell/shell.cpp [hunk 3]
```diff
@@ -714,6 +714,21 @@ void SHELL_Init() {
...
 	MSG_Add("SHELL_CMD_DELETE_HELP","Removes one or more files.\n");
+	MSG_Add("SHELL_CMD_DELETE_HELP_LONG", "Usage:\n"
+	        "  \033[32;1mdel\033[0m \033[36;1mPATTERN\033[0m\n"
+	        "  \033[32;1merase\033[0m \033[36;1mPATTERN\033[0m\n"
+	        "\n"
+	        "Where:\n"
+	        "  \033[36;1mPATTERN\033[0m is a file name or pattern that contains wildcards (* or ?),\n"
+	        "          such as \033[36;1mfile.txt\033[0m, or \033[36;1m*.bat\033[0m.\n"
+	        "\n"
+	        "Notes:\n"
```
Comment (kcgen) [inline src/shell/shell.cpp:725]: Instead of Notes:, let's use Warning:

File: src/shell/shell.cpp [hunk 4]
```diff
@@ -714,6 +714,21 @@ void SHELL_Init() {
...
+	        "  Be careful with deleting files containing wildcards, especially \033[36;1m*.*\033[0m.\n"
+            "  All files matching the pattern(s) will be deleted.\n"
+	        "\n"
+	        "Examples:\n"
+	        "  \033[32;1mdel\033[0m \033[36;1mtest.bat\033[0m\n"
+	        "  \033[32;1mdel\033[0m \033[36;1mc*.*\033[0m\n");
```
Comment (kcgen) [inline src/shell/shell.cpp:731]: Can we include an example with a ?, And maybe one with ? and * (if we have room?)

File: src/shell/shell.cpp [hunk 5]
```diff
@@ -714,6 +714,21 @@ void SHELL_Init() {
...
+	        "  \033[32;1mdel\033[0m \033[36;1mPATTERN\033[0m\n"
+	        "  \033[32;1merase\033[0m \033[36;1mPATTERN\033[0m\n"
+	        "\n"
+	        "Where:\n"
+	        "  \033[36;1mPATTERN\033[0m is a file name or pattern that contains wildcards (* or ?),\n"
+	        "          such as \033[36;1mfile.txt\033[0m, or \033[36;1m*.bat\033[0m.\n"
+	        "\n"
+	        "Notes:\n"
+	        "  Be careful with deleting files containing wildcards, especially \033[36;1m*.*\033[0m.\n"
+            "  All files matching the pattern(s) will be deleted.\n"
```
Comment (kcgen) [inline src/shell/shell.cpp:727]: This line is indented with only spaces (instead of one tab and then aligned with spaces).

**Review by kcgen — CHANGES_REQUESTED** [review]
Looks good @Wengier!

Just a couple minor suggestions.

File: src/shell/shell.cpp (hunk 1 above)
Comment (Wengier) [reply src/shell/shell.cpp]: Done.

File: src/shell/shell.cpp (hunk 2 above)
Comment (Wengier) [reply src/shell/shell.cpp]: Done.

File: src/shell/shell.cpp (hunk 3 above)
Comment (Wengier) [reply src/shell/shell.cpp]: Done.

File: src/shell/shell.cpp (hunk 4 above)
Comment (Wengier) [reply src/shell/shell.cpp]: Done.

File: src/shell/shell.cpp (hunk 5 above)
Comment (Wengier) [reply src/shell/shell.cpp]: Thanks for catching this single line. Fixed.

**Comment by Wengier** [comment]
Thanks @kcgen for reviewing. I have hopefully addressed all suggestions you made. Please check it out.

**Comment by kcgen** [comment]
Thanks @Wengier!

**Pull request merged**